util/dvsim/examples/testplanner/foo_testplan.hjson

```diff
@@ -43,8 +43,8 @@
             the description supports Markdown formatting. Add things:
             - like bullets
             - something in **bold** and in *italic*
-              - A sub-bullet item<br>
-                Continue describing above bullet on a new line with a HTML line break.
+              - A sub-bullet item \
```

Comment:
Does it work to remove the space before "\"?

Reply:
Done.